miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/SessionConversationIdTag.java

```diff
@@ -66,12 +68,12 @@ protected int writeTagContent(TagWriter tagWriter) throws JspException {
     String conversationId = (String) pageContext.getRequest().getAttribute(attributeName + "_cId");
 
     // if no value was found then try to pull value as request parameter.
-    if (conversationId == null || conversationId.trim().length() == 0) {
+    if (isStringBlankOrNull(conversationId)) {
       conversationId = pageContext.getRequest().getParameter(attributeName + "_cId");
     }
 
     // if a conversation Id was found then process it.
-    if (conversationId != null && conversationId.trim().length() > 0) {
+    if (!isStringBlankOrNull(conversationId)) {
```

Comment:
No, wait, that changes the flow of this code...I'm not sure what it is supposed to do.